docs/DeveloperGuide.md

```diff
@@ -32,7 +32,7 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
```

Comment:
Just leaving a note for this, the changes to the URL would cause it to break since it's linking to SE-EDU's AB3 page, so we should remember to fix this when we update our DG proper.